packages/template-retail-react-app/app/components/link/index.jsx

```diff
@@ -8,26 +8,26 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import {Link as ChakraLink} from '@chakra-ui/react'
 import {Link as SPALink, NavLink as NavSPALink} from 'react-router-dom'
-import {buildPathWithUrlConfig} from '../../utils/url'
-import useSite from '../../hooks/use-site'
-import useLocale from '../../hooks/use-locale'
+import {useSite} from '../../hooks/use-site'
+import {useLocale} from '../../hooks/use-locale'
+import {useAppConfig} from '../../hooks/use-app-config'
 
 const Link = React.forwardRef(({href, to, useNavLink = false, ...props}, ref) => {
     const _href = to || href
     const site = useSite()
```

Comment:
Maybe we can call this siteContext, so it won't have a duplicate with the `site` as the property inside. Same for locale